runtime/interpreter/interpreter.go

```diff
@@ -3056,6 +3063,12 @@ var converterFunctionValues = func() []converterFunction {
 
 		addMember(sema.FromStringFunctionName, fromStringVal.hostFunction)
 
+		if declaration.nestedVariables != nil {
+			for name, variable := range declaration.nestedVariables {
```

Comment:
Not sure about the lint error upon range statement over map. Do you have suggestions?

Reply:
When iterating with a `range` over a Go `map`, the iteration order is non-deterministic. That may cause problems in Cadence, if the loop body has side-effects that may impact the execution result (execution results must be fully deterministic).

Though in this case the body is OK (it only mutates another map), and we could ignore the lint error here, it might be better to just ignore the non-determinism of the loop all-together. You could make the iteration order deterministic by just changing the type of field `nestedVariables` to `[]struct{Name string; Value Value}`, i.e. a slice of key-value pairs.

Reply:
Thanks, updated!